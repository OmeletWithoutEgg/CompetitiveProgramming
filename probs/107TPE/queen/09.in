248
15147282 67740293
94259847 100952393
-19841691 40984176
86147284 15526756
91473303 102859379
-9199803 79696129
-19836704 103966533
-29339396 31989452
114549852 63740495
106013108 32584256
112724663 46944961
47567720 13673362
-9161509 79428567
-11483858 88997859
113056630 48166534
49355142 13061813
-77402945 26117826
-99478896 101377090
-81020047 27748703
-9182542 79576802
65912701 110650746
78201721 12539924
31277299 97904390
71287734 11110137
-64002196 122193948
-86557258 113983146
-31328023 114336311
-9550847 63166680
18900314 80852899
-95633236 106155881
-107715984 82229873
-60359160 122450095
79047509 108547424
-31919286 30240148
-10969017 57510682
108836508 37277543
-12817288 92402500
30894280 23776072
78477118 12618355
28266594 26364442
-80494930 27490220
-75393864 25356071
48199768 13448474
98853363 97132543
-104308357 52040770
106934836 33992496
-43230009 120032798
-107456567 83454171
21084674 36119963
58812305 11008134
73677943 109844018
20103677 37948503
-30492194 31178709
19655298 38849885
-108133192 65130174
93815701 101275160
-29517015 31861349
-87137773 31369627
-38488366 26735069
-21914641 38587509
-44763086 24453147
-51675350 22970962
-102179622 97124816
83059641 107152095
17352023 44436318
114476837 64757042
-68531722 121497476
-108305907 66405285
-9420744 81075470
-107311737 84079103
52722776 12108884
57384734 110136997
27136421 27604011
49912163 12886496
113719485 51094083
-74776433 119815303
109043537 83660704
-98501406 42246895
107482076 34880503
79706887 12989566
-64110008 22774579
112047110 76572322
-11398539 88752070
-107408892 61292910
40516138 104459827
-73151396 24619616
21877978 86571116
-71069296 24038685
-81488113 116971266
15920101 71889186
-68978083 23551246
51421186 108886604
42271378 15951656
103621203 29348948
-89170812 32854282
-100900751 45700088
112022352 76645918
89360577 104130743
-80078499 27290625
82068819 13801174
-97595814 41089633
-22807694 107313137
19912581 38327116
114446372 56221039
93187778 101718953
19698475 82572598
-21177673 105552095
114463083 64921291
-96101311 39321918
-8676169 72400956
105933485 88866351
-9792560 61971350
14928722 65991292
-99922639 100740130
-37485845 27190763
113697278 50980936
-24477640 36002893
112753611 47046800
-16969078 44900837
89662289 17375889
-53037676 122159493
-103453118 50229397
113617836 50586949
16555367 74357833
-60176781 22500955
-107183250 84605146
-12203876 90926507
113029673 48062597
109431965 82894641
19195592 39824463
56381256 11354359
107454164 86499477
19550841 82267055
-80724892 27602477
-101088832 98957873
-22580972 37878702
18198754 79181706
15897724 71791583
73798976 109821631
-82490447 116442930
87718307 105024422
-64514775 22820500
-57189372 22500538
20508869 37171015
-105725131 89401476
-64834019 122097782
-91048088 110584268
-104675088 92076744
-10620571 86286590
-16309399 45925427
114603406 62691490
15135050 53679510
41631274 16277699
82253697 13870326
96293651 99375049
55814548 109880993
-13011436 52113320
80911354 13386958
112720462 46930249
86239325 15570716
-72525850 120522000
-64053665 122188407
76922776 12197856
-107901224 63709850
-106850073 85867574
-102465310 96613586
72428755 11276511
-41373755 25567572
-40116223 26050737
19892599 38367172
-23452056 36992290
-63148755 122277982
93542009 19863292
42117404 16029028
54178838 11774384
-28213286 32829767
-32800621 115262295
-103191993 95245423
-86077746 114301273
105281280 31535318
15777123 71249259
114130738 53518136
-51660698 121983823
64899182 10667484
21356303 35644059
-108640651 70595747
-27866898 33098326
-49857657 23262224
40545703 16857565
-99174199 101802560
29074245 95806029
52176282 109087343
-101457952 46599599
65111960 110664648
-11573324 55705603
16846943 75343401
93221227 19637993
-94053656 37145313
-86234127 30758463
-41582399 119465716
-90833210 34191084
16801400 46139417
-20592055 40080823
114526075 57228809
107434843 34802201
-101186979 98800016
114597374 58498438
76274366 12038198
114528997 57271466
31687493 23065754
98419425 97534825
90822807 18067635
-21767951 38747318
95820330 21576396
-85808056 114476732
-100882616 99285345
-43396307 24870366
-10925765 87307612
15309204 52540527
-25759534 110114837
-8763338 69526251
88489032 104614921
67458134 10746067
-99565667 43702675
58955883 10991482
113428891 49708961
105605960 31993644
114644355 60744451
-52665253 22841048
-10552205 58910458
79033564 12782052
50105527 12827304
-65115467 22894817
87961757 104896933
-107771130 81948323
-93069846 36186959
-11543541 55789480
74015312 109780848
-73508832 24729185
-20625497 104915306
68088376 10785584
31482610 23246323
-57032436 22505453
112213524 76067481
-88578917 112551139
-102591167 96383819
-17022039 44820907
37294768 102523695
