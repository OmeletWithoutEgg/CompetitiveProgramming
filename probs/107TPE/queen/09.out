2852010.18 66377581.05
