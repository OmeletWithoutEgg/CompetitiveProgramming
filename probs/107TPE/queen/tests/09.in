660
80252657 84954828
127285855 64322475
57964512 59887726
125847534 70985707
125622221 58444065
118874722 51123383
107307532 84121996
-95733814 -70708242
71016198 81945372
66631995 79584801
-85311041 -71785051
-76451927 -35850264
-106949558 -43802467
103844832 44881507
-71993779 -63481925
-104244138 -39379029
-86706723 -71958974
107291298 45744642
-107787812 -46030168
126675682 60944522
-102583918 -66344948
115559302 80797966
-81230515 -33219238
68482787 49192083
-108675421 -52550685
73774956 46793190
85945717 43998368
-84806341 -71697712
72237681 82475547
-70104890 -43735355
84424138 85687565
-74531962 -37471348
58545697 58620509
-106230552 -42350317
-107676702 -58221666
-93844065 -71325198
115237357 80969741
-100150102 -68359478
-106576096 -60892911
-107778891 -46001451
-68540348 -49128233
-72339812 -39945919
63120001 76960244
125058426 72407979
62970347 53044200
93306583 43678888
-97618871 -69876139
64568032 51724006
126404779 60166261
-102512991 -37491066
-68665980 -48355645
-106964082 -60069314
93647763 86184534
-75532234 -36575715
-83317942 -71362236
-97336843 -70015485
125515263 71625715
-69824946 -59516584
75601416 83709304
-68517127 -54608278
71264525 47814264
-107658318 -45627322
-72199954 -63769418
-82236769 -71042848
88513772 43779709
71006201 81940832
109382072 83469750
57838975 69662073
126148101 70335451
-68346397 -51446744
117504673 79662931
-68356901 -51130345
-75523435 -67321357
-68696540 -48192265
119227041 78494501
82868976 85453752
125476096 58174557
-82343052 -71077168
96323711 43818723
-73521618 -65393605
-69033580 -46730166
-104199447 -64580662
82479864 44483261
120489888 52356686
71162500 82011426
97762525 43941501
-101280624 -36401089
89224819 86131662
67863093 80330535
121389636 53134321
-93795370 -32566019
-88889324 -72036083
-101935608 -36958008
-78502981 -34512069
-107986888 -57188743
90232361 86173366
78769141 45261454
114104050 48329174
118339416 50753088
90550907 86182899
-73365515 -65218938
75770354 46107990
-68534887 -49166800
-71983531 -40437089
103326447 85097825
126543970 69323241
116151384 80470734
-88905139 -31868659
58318093 59081479
-68587791 -55090985
69259208 81093283
126794669 68529777
58749766 58236512
-91834505 -32139159
-92751906 -32313529
-85104497 -71750897
111108844 82841898
74325796 46593544
125353514 57957357
57029705 63593076
58196001 59345790
106728802 84285528
-107447374 -45026185
84931144 44117287
56966155 65347468
126882288 68205739
72670475 47219126
-101058733 -36223020
88686290 86102146
127280589 64220534
-87982543 -72032668
-88240939 -72037777
-73327482 -38728686
-90587292 -71933028
-73572186 -65449318
-102111411 -37115927
125412711 58061292
-84210353 -71577511
99249215 85764061
-108647014 -53174223
84869258 44125149
-73336826 -65186387
-102352839 -66565449
125659062 58513990
64553762 78136430
74983674 46365742
83347459 85530547
-106473315 -42808504
64109473 77789731
-107680247 -58210911
61083208 54942826
64010281 52161060
-84502143 -71638688
75120030 46319939
124577599 56730113
103607128 85040027
-68425241 -53796817
122158818 76007904
115443088 49010871
74934381 83488913
-70806325 -61576947
-72065858 -63583653
-88793233 -31866797
-94348759 -32724041
-107965490 -57267024
-95155739 -70918933
-98579905 -69358949
-77213979 -68593255
127083317 67295509
127300277 64864252
-69083719 -57356318
-97816734 -69775085
-105111521 -40538332
72788284 82699320
119529324 51600835
88079133 86062800
-86386886 -71927794
-95479200 -33100943
57019872 66180690
122532352 54242701
112269999 82371892
112736782 47700051
-70359832 -43190986
58611729 58493491
59113384 57607106
-91752006 -71778825
-76577487 -68146681
62420383 76318418
61480033 54504453
-92320688 -32226105
127299576 64783539
63793263 52337651
68528180 49167192
-96393631 -33461501
102699236 44650078
-71852168 -63278080
-78051299 -34776934
-104041368 -39131503
-68581186 -48855319
-99855594 -68562744
-103646622 -65232790
-87125206 -31912386
109967095 46605221
106584552 84325091
56974877 64308071
100354485 44256686
57234041 62283673
-103470007 -38474695
-108529602 -54435783
-71638681 -40944215
-77707369 -34989381
116460972 80293532
-68851662 -47456725
123647503 55497026
-96864402 -70236872
-89890321 -71992661
77970518 84404540
118596847 50929008
-103841253 -38894879
-107114194 -44182301
-105869834 -41717822
-72012434 -63508383
-81385378 -33160274
-99628694 -68714167
-81624364 -70832328
97475543 43914079
69232500 81079444
88319669 86079159
-96798078 -70266770
-97290918 -70037662
-74629590 -66525931
-91829980 -71766034
-75665312 -36465288
96592815 43838904
-105886497 -62158544
94595559 86152022
-69737480 -44605762
94749700 43726065
-96027169 -70593614
66800670 79691425
-94442195 -32752411
-105675968 -41398581
126916404 68070279
-108671359 -51230723
73996388 83159391
117977105 50511975
62624529 76510977
56960444 65086371
-95768779 -70694855
-87265581 -72001199
-68412720 -53656220
113740988 48155843
115037172 81074444
-70428884 -60853927
67752280 80266377
-69628688 -59016773
103257878 85111702
-108630413 -53420527
-103814394 -38863676
-107221946 -59461214
77205969 45676760
58861194 58036786
61496877 54486373
57636065 60784183
104208111 44960471
62298052 53670526
-103122783 -38102151
66970754 79797456
57065153 66582517
104736191 45080153
77773227 45519702
126258867 70073489
-108639014 -53298573
-78625067 -69461291
86514719 85931516
127088705 67266143
-68944282 -47072431
96155120 86064602
62819764 76690968
-108670760 -51214301
-71947257 -63415550
97057379 85994584
-107600301 -45455763
-73972880 -65876469
-82937332 -32647188
-87512449 -72014909
-70056985 -60061831
-82707103 -71189847
82762789 85436116
-88269357 -31866303
-108181359 -47494341
97937088 85912431
58686853 58352261
-99908908 -68526516
72416133 47322258
-98419860 -34454668
126650180 69006703
-94416008 -71160031
-87326702 -72004875
82501327 85391769
70576528 48128829
61667349 75565647
-71432295 -62639947
75766747 83762194
73602831 46857286
68579493 49139160
126295669 69983250
-73624777 -65506819
-95621289 -33153620
-88475190 -72039560
111302522 47105046
-98105802 -34281978
-69557640 -45078928
127260568 63922751
-108419351 -55197663
114150157 81519808
-72269834 -63864483
114794277 81199372
-82766032 -32697059
-79719328 -70030849
-97740610 -69814288
-70539195 -61072771
113324838 47962838
-93216058 -71485829
-78362248 -34592911
112596431 47639003
-80749125 -33411913
-71483449 -62720583
-94758071 -32851988
-107857018 -46258331
58111769 59536266
-79525194 -69935939
58838591 58076777
123122779 54881655
-92936379 -32353920
-98704185 -34617358
-74343236 -66250045
-87155354 -31910340
88436256 43784607
-71493784 -62736772
123056449 54807236
73345095 46954815
-105491383 -62797897
69004086 80959934
94781532 43727512
-101548227 -36622827
-108359447 -48361351
-93285026 -32435208
-82807878 -32684725
113512939 81821992
60648749 74418401
-102563704 -37539922
-68382127 -50654841
-104365765 -39531442
101857118 85374735
-94408388 -71162352
95592303 43770292
123746942 74251891
-68353637 -52690456
-96650938 -33572362
-72151130 -63702298
-88136715 -31868328
-81936186 -70942259
-87052413 -31917514
58559175 58594356
-70860488 -61675098
115669497 49133140
-73477693 -65344869
118131363 79257606
85039888 85767687
66640765 50280949
82646742 85416595
107274282 84131602
125667848 58530789
-87663150 -72021793
-78960850 -34259287
80516884 85011440
-107842767 -46210542
-68852206 -56450076
-99888744 -68540247
-69485058 -45280753
71971439 47507301
-108374466 -48444665
-69782792 -59412487
-99989137 -68471529
73649034 46839999
62567868 53413344
113095275 47858890
-68361023 -51035577
-73948131 -38053642
78048396 84425186
64955109 51433956
-84533481 -71644992
122323391 75843004
117001414 79973609
-75020511 -37018887
-69403352 -45516484
-72960425 -39158687
61364007 54630119
-104076321 -64730839
77928549 84393358
57043856 63464009
56966505 64513049
89113441 86125971
126116488 70407766
119310011 78433633
127122604 62799328
-106000718 -41941207
76225709 45965846
68379545 49249019
-72752445 -64490839
127003165 67695304
126875744 61640260
-68561481 -54920639
-73817421 -65713694
-71029312 -61973006
61826701 54140405
58899475 57969642
58841249 58072061
90972117 86192818
-104615338 -64050449
-92771989 -32317839
70639085 81771772
-69694042 -59187632
-108298443 -48041742
70107565 48352647
-75348653 -36731256
74181882 83226423
84410078 85685654
110516660 83066443
59731521 56659491
-96464395 -70412872
-74407172 -37591847
84339583 85676017
114453904 81370629
-100207169 -35585250
122711869 54431575
-68346509 -52462108
96647975 43843198
-75529060 -67326069
-70332701 -43246945
65548959 78862991
123150060 74958861
126853307 68316450
-102424672 -37406789
107428472 45784493
119589917 51646494
-80782850 -70506495
70626436 48105466
127299425 65100083
-82591808 -32749468
104217354 44962516
-76039644 -36164776
-96299381 -70482495
65803486 50832649
-71517449 -62773714
-83388449 -71380810
56960085 64819665
124328269 73492855
-107804222 -57821044
68162364 80501042
-68736213 -55913888
126937731 67982466
61675624 75574315
-108217292 -56249864
80759181 85062098
126988221 67763503
91264637 86197908
71743076 47604715
-99287302 -34970649
-97548126 -33992834
122294436 75872263
-95027217 -70963090
-68345994 -51462940
-100132244 -35532417
-98770762 -34656344
-76718183 -35655869
-80698738 -33432913
-100647358 -35906162
-107904286 -57484371
-70442699 -43022737
127229849 63588469
-77071167 -35408224
-102626224 -66303812
-108665168 -51075831
-106100149 -42115455
-83822794 -71489278
57090598 66769913
-92053675 -71727602
127252805 63828673
76012246 83839483
98126239 85892960
-107651741 -58296826
70860404 81874242
-72351466 -63974072
-85901192 -32033826
-68375189 -50766405
125860250 58911601
60811917 74613969
-72569921 -64259907
-101963174 -66921917
59751151 56631445
61799875 75703341
-68399380 -50412202
-90574531 -71934338
99868108 85683182
83325907 44344149
-68342985 -51607207
70273711 48272456
-77688052 -35001597
-70853624 -61662730
-69493123 -45257988
81474759 44666545
65869925 79083907
-81437357 -33140808
121971226 76191850
-68367383 -50906114
-72288987 -40014105
122399965 75765111
80475659 44868629
-78879057 -69601127
-70822492 -42298061
-74151042 -66058619
-108508083 -54602107
-106208525 -42310122
-73008987 -39100376
124753159 72882577
-107756605 -57974072
-105104947 -40528857
-106338684 -42550878
111631841 82635196
-71369865 -41364071
-79703252 -33881354
-69338267 -58193052
61858021 54108317
-97715957 -34077544
105923256 84500459
-108674570 -51325968
-92380359 -71666819
125145002 72266945
65563232 78872947
-79566822 -33947934
-102383460 -37367811
59380554 57181092
127272120 64081791
-90786705 -71911493
-71736670 -40796694
-101257034 -67522529
-108627006 -50438334
-88513369 -72039593
-69906552 -44190946
-71864202 -40608828
-70269242 -43379598
94064666 86172145
63359473 52702488
-96103766 -70562813
-69648748 -59069717
-69907368 -59715434
-105662441 -41376798
-68340013 -51940415
-81585347 -33086273
78331103 45372336
-90097281 -71977483
73575668 83003855
-108188401 -47525263
126871260 61623017
86285087 85908585
80056958 44959359
115392809 80887326
-106159338 -42221139
57804643 60301289
126782026 68574090
60175353 73820495
-68563200 -48972368
73851484 83106374
73130493 47037462
-106124770 -42159236
-103827005 -65026130
-103862635 -38919812
59014716 72099572
-102431309 -66491354
-79199893 -69771067
64546834 51740195
57439398 68436947
121125062 76973445
106024178 84474327
115202763 48883382
79176579 84709270
120120397 52057586
93834360 43691971
72600843 47247168
126712426 61062639
79325614 45126595
63877257 77602480
94415608 43711942
122523271 54233264
70595452 81751371
122881261 75257320
56989950 65819620
58910435 71920780
104415380 45006750
97495156 43915907
62122979 76029458
77216133 45673881
120115750 52053894
112702578 47685115
-77112275 -35380108
-108629775 -50475269
95923085 86080295
-72222790 -63800613
79262103 45141651
-74064715 -65970936
106063371 45407214
-87016849 -31920115
57481218 61286090
98335492 44000659
-88368816 -31865355
125233634 72119297
-70272109 -43373551
-70582002 -42748501
-108364241 -55516717
-76955757 -35487957
126780050 68580967
85475577 44051148
126378340 60097303
124985528 72524419
57990619 70047791
111507821 47186392
70332324 48244403
-107164070 -59602656
-97771367 -69798498
60877846 55179737
-108213119 -47635650
-98709236 -34620303
124134684 56116896
60966952 55076051
-69791304 -44470800
-103726770 -65141669
59781517 73283052
65080069 78528830
-98506296 -34503480
-69450051 -45380594
-68508895 -54546085
76280592 45949055
100008596 44207494
64020204 52153076
107177425 84159437
-108585032 -53943338
-77937785 -34846002
99034257 85790508
-92080059 -71722898
109506683 83427129
111151418 47045961
66193463 50570828
-106374816 -61285310
123395262 55195065
-91387146 -32069903
-74220081 -37776464
-84256586 -32316938
-102898557 -66033120
