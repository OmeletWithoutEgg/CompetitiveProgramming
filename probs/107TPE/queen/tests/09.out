-25873419.00 6700251.00
